app/routes.py

```diff
@@ -96,8 +92,10 @@ def colorized_icon(
         if image.mode == 'P':
             image = image.convert('RGBA')
 
-        new_size = calculate_icon_size(image.size, scale)
-        image = image.resize((new_size[0], new_size[1]))
+        #As of now all scaling will only be done on the front-end. Since all icons are stored in the
+        # size of 48 x 48, a resizing is not necessary
+        #new_size = 48
+        #image = image.resize((new_size[0], new_size[1]))
```

Comment:
Those 2 lines can be deleted I guess

Reply:
What about for request with scale bigger than 1, the service should resize (even though currently not used).